src/main/java/echoquery/sql/QueryResult.java

```diff
@@ -42,15 +48,45 @@ public String toEnglish() {
       return null;
     }
 
-    StringBuilder aggregation = new StringBuilder();
     StringBuilder translation = new StringBuilder();
-    AstVisitor<Void, Boolean> translator =
-        new DefaultTraversalVisitor<Void, Boolean>() {
+
+    /**
+     * Visit each node in the tree and convert it to natural language.
+     * The capture boolean signifies whether or not certain components of the
```

Comment:
Oops removing this outdated comment, we have enough metadata to not do weird things such as not capturing parts of the tree